config/config/wc-config.yaml

```diff
@@ -92,17 +92,6 @@ falco:
     affinity: {}
     nodeSelector: {}
 
-## Elasticsearch cluster topolgy.
-## Used in prometheus alerts.
-elasticsearch:
```

Comment:
No longer used or?

Reply:
It was used to get the node count into wc/user alerts, but the alerts was turned off.